### DISCUSSION TITLE
Zeek Intel Not working

### DISCUSSION THREAD

**Discussion by bdorr1105** [Security-Onion-Solutions/securityonion#3830] [Unsupported Versions] [answered]
I am unable to get Zeek Intel working. I checked the reporter log and it is blank. I looked to see if the intel.log file was being generated and that is also missing. I added 8.8.8.8 and ran a test ping to it and zeek is not generating any hits. Please help point me in the correct direction. I saw another post mentioning in relation to this, but not directly this. is Zeek Intel working properly? Thanks.

![seconion2-intel-dat](https://user-images.githubusercontent.com/12386911/114306660-7308b380-9aaa-11eb-8b61-6cc68ebad4e7.PNG)

**Comment by dougburks** [comment, marked as answer]
I've partially duplicated this and created Issue https://github.com/Security-Onion-Solutions/securityonion/issues/3850
